lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
+    tmp_path = File.join(FileUtils.pwd, 'tmp')
+    if delete
+      FileUtils.rm_rf tmp_path
+    end
+    FileUtils.mkdir_p tmp_path
+    tmp_path
```

Comment:
Having seen https://github.com/opf/openproject-translations/pull/17/files#diff-417e4dc2a9a0bd9f6607edfca11a2304R18 now, I'd say it is a good idea to create a `within_tmp_directory` method that accepts a block, creates a tmp directory, yields, and removes the directory.